teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorProposerOptions.java

```diff
@@ -91,6 +91,16 @@ public class ValidatorProposerOptions {
       hidden = true)
   private UInt64 builderRegistrationTimestampOverride = null;
 
+  @Option(
+      names = {"--validators-builder-registration-pubkey-override"},
+      paramLabel = "<BLSPublicKey>",
+      showDefaultValue = Visibility.ALWAYS,
+      description =
+          "Set a constant pubkey in BLS format to be used in validator registrations against builder infrastructure.",
+      arity = "1",
+      hidden = true)
+  private BLSPublicKey builderRegistrationPubkeyOverride = null;
```

Comment:
does this need to be string and then converted in type BLSPublicKey

Reply:
Yeah, it is cleaner to be a string and then we do the conversion in the `ValidatorConfig` . I made this change as part of the PR i raised in your fork.

Regarding the open question, let us know when you have more clarity and we can think about the implementation.